### DISCUSSION TITLE
text screen suggestion

### DISCUSSION THREAD

**Discussion by theAC130onsteam** [wiremod/wire#3223] [Suggestions]
perhaps make it so you can have even more text screens? it would be great as i need it for a project i'm working on, thanks

**Comment by Denneisk** [comment]
What do you mean exactly? You can change the limit using the convar `sbox_maxwirescreens`.

**Reply by theAC130onsteam** [reply]
As in, raise the amount of render targets being able to use or something, (might not be possible sadly, no idea if i can)
